src/main/java/com/stories/swaggerConfig/Config.java

```diff
@@ -0,0 +1,24 @@
+package com.stories.swaggerConfig;
+
+import org.springframework.context.annotation.Bean;
+import org.springframework.stereotype.Component;
+
+import springfox.documentation.builders.PathSelectors;
+import springfox.documentation.builders.RequestHandlerSelectors;
+import springfox.documentation.spi.DocumentationType;
+import springfox.documentation.spring.web.plugins.Docket;
+import springfox.documentation.swagger2.annotations.EnableSwagger2;
+
+@EnableSwagger2
+@Component
+public class Config {
+	
+	@Bean
+	public Docket SwaggerApi() {
+		return new Docket(DocumentationType.SWAGGER_2)
+				.select()
+				.apis(RequestHandlerSelectors.basePackage("com.stories"))
+				.paths(PathSelectors.any())
+				.build();
+	}
+}
```

Comment:
Why are you adding Swagger in this sprint?

Reply:
For try response of the PUT method

Reply:
Swagger implementation is not required for this Sprint, please remove it.